content/it/docs/concepts/overview/components.md

```diff
@@ -0,0 +1,124 @@
+---
+title: I componenti di Kubernetes
+content_template: templates/concept
+weight: 20
+card: 
+  name: concepts
+  weight: 20
+---
+
+{{% capture overview %}}
+Facendo il deployment di Kubernetes, ottieni un cluster.
+{{< glossary_definition term_id="cluster" length="all" prepend="Un cluster è">}}
+
+Questo documento describe i diversi componenti che sono necessari per avere 
+un cluster Kubernetes completo e funzionante.
+
+Questo è un diagramma di un cluster Kubernetes con tutti i componenti e le loro relazioni.
+
+![Components of Kubernetes](/images/docs/components-of-kubernetes.png)
```

Comment:
nit: `alt` attribute is not translated (but also, nor is the image).

Most localizations have not yet localized images.